systemtest/src/main/java/io/strimzi/systemtest/resources/specific/OlmResource.java

```diff
@@ -67,7 +67,7 @@ public void create(String namespace, OlmInstallationStrategy olmInstallationStra
 
     @Override
     public void delete() {
-        this.deleteOlm(deploymentName, namespace, csvName);
```

Comment:
What is the point of removing `this` reference? Did you face some issues with that?

Reply:
Yes I did have problems with accessibility of variables. See more changes in that file for example.